apps/source_academy/priv/repo/migrations/20170626081409_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule SourceAcademy.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :email, :string
+      add :is_admin, :boolean, default: false
```

Comment:
Change this to :role to reflect frontend
Type = string
Allowed values = staff, student, admin
Default to student, admin can simply elevate it to staff

Don't forget to validate in changeset.